integration_tests/suite/test_calls.py

```diff
@@ -131,6 +131,108 @@ def test_given_some_calls_with_user_id_when_list_calls_then_calls_are_complete(s
             )
         )))
 
+    def test_call_direction_internal_internal(self):
+        self.ari.set_channels(MockChannel(id='first-id'), MockChannel(id='second-id'))
+        self.ari.set_channel_variable(
+            {
+                'first-id': {'WAZO_CALL_DIRECTION': 'internal'},
+                'second-id': {'WAZO_CALL_DIRECTION': 'internal'},
+            }
+        )
+        self.ari.set_bridges(MockBridge(id='bridge-id', channels=['first-id', 'second-id']))
+
+        calls = self.calld_client.calls.list_calls()
+
+        assert_that(calls, has_entries(items=contains_inanyorder(
+            has_entries(call_id='first-id', direction='internal'),
+            has_entries(call_id='second-id', direction='internal'),
+        )))
+
+    def test_call_direction_internal_outbound(self):
+        self.ari.set_channels(MockChannel(id='first-id'), MockChannel(id='second-id'))
+        self.ari.set_channel_variable(
+            {
+                'first-id': {'WAZO_CALL_DIRECTION': 'internal'},
+                'second-id': {'WAZO_CALL_DIRECTION': 'outbound'},
+            }
+        )
+        self.ari.set_bridges(MockBridge(id='bridge-id', channels=['first-id', 'second-id']))
+
+        calls = self.calld_client.calls.list_calls()
+
+        assert_that(calls, has_entries(items=contains_inanyorder(
+            has_entries(call_id='first-id', direction='outbound'),
+            has_entries(call_id='second-id', direction='outbound'),
+        )))
```

Comment:
All those extra scenarios are redundant with the unit tests that compute the direction has the correct value. I think only one integration test would be enough.